pages/common/plesk.md

```diff
@@ -0,0 +1,28 @@
+# plesk
+
+> Plesk hosting control panel CLI interface.
+> More information: <https://docs.plesk.com>.
+
+- Generate an auto login link for the admin user:
```

Comment:
```suggestion
- Generate an auto login link for the admin user and print it:
```
Only apply if what I'm say is correct ;)

Reply:
Yes, you're right :)